### DISCUSSION TITLE
Welcome to logicanalyzer Discussions!

### DISCUSSION THREAD

**Discussion by gusmanb** [gusmanb/logicanalyzer#13] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by samadhan-babar** [comment]
sir can you give version 6.0 PCB file , i have interested 
i have one question 
how can we connect  logic analyzer to computer using wifi (internet ) , i main using ip address in software
can you explain brief

**Reply by DeeJay** [reply]
I am just another interested user, but perhaps I can help with this part of the question: _"how can we connect logic analyzer to computer using wifi"_

There is a step-by-step description in the README from when this feature was introduced - [https://github.com/gusmanb/logicanalyzer#update-31012023---pico-w-support-and-wifi](https://github.com/gusmanb/logicanalyzer#update-31012023---pico-w-support-and-wifi)

**Reply by samadhan-babar** [reply]
thanks

**Reply by samadhan-babar** [reply]
do you have this 6.o version pcb file and , if you have can give me

**Reply by gusmanb** [reply]
V6 is still not finished and PCB files are not public as I'm actively working on the dessign and I already had problems with users that downloaded the V6.0 branch and ordered non-functional PCBs.
